docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -263,27 +263,40 @@ The changes will take effect at the next launch.
 
 ## Troubleshooting
 
-Logging out of a cluster, closing the app and logging in again resets all app state related to that
-cluster. This can help if you encounter a bug which renders the user interface partially unusable.
-It might also help if you have issues with connecting to an active cluster that don't happen in the
-Web UI.
+If you encounter a bug which renders the UI partially unusable, resetting app state related to a
+particular cluster might help. To do this, log out of the cluster, close the app and log in again.
+
+To completely wipe all app state, close the app and remove the internal `tsh` folder and the
+`app_state.json` file. This will log you out of all clusters and clear all remembered tabs and
+connections.
```

Comment:
```suggestion
If you are unable to connect to a cluster or resource using Teleport Connect, try resetting the app state for the cluster. 
To reset the app state:

1. Log out of the cluster.
1. Close Teleport Connect.
1. Open Teleport Connect, then log back in to the cluster.
```
What does "partially unusable" mean? What is the symptom the user sees? What does "app" refer to (it seems like "app" refers to Teleport Connect but I'm not sure)? Same question for the next paragraph. I took at swing at it, but I'm not sure if this is what you mean.

```suggestion
To completely wipe all app state:

1. Close Teleport Connect.
1. Remove the internal `tsh` folder and the `app_state.json` file to log out of all clusters and clear all remembered tabs and connections.
```

Reply:
When I originally wrote this section, I was thinking of issues like gravitational/teleport.e#858. So what "partially unusable" really means is that due to actions performed outside of the app, the UI of the app is partially broken and the only way to resolve the issue is to reset app state, either in full or just for a specific cluster.

I don't really know how to put it into words so that it's clear the user shouldn't attempt to do this for all errors.

I think this advice to log out of a cluster or wipe all state would only be useful if some UI code crashes. The symptoms that could be seen by the user would be blank window or "Internal Error" screen. This would typically happen if the app gets into some unforeseen state, for example due to server-side changes as I mentioned or due to the downgrade of the app.

And yeah, "app" refers to Teleport Connect.

I'll try to rewrite this with what's discussed here in mind, thanks!